pkg/placement/membership.go

```diff
@@ -258,7 +258,9 @@ func (p *Service) processRaftStateCommand(stopCh chan struct{}) {
 }
 
 func (p *Service) performTableDissemination() {
+	p.streamConnPoolLock.RLock()
```

Comment:
If we use `sync.map` to assign streamConnPool, can we remove those lock actions here and below?